src/Lightweight/DataMapper/DataMapper.hpp

```diff
@@ -183,6 +183,40 @@ class DataMapper
     template <typename Record, typename... InputParameters>
     std::vector<Record> Query(SqlSelectQueryBuilder::ComposedQuery const& selectQuery, InputParameters&&... inputParameters);
 
+    /// Queries multiple records and returns them as a vector of std::tuple of the given record types.
+    /// This can be used to query multiple record types in a single query.
+    ///
+    /// example:
+    /// @code
+    /// struct Person
+    /// {
+    ///   int id;
+    ///   std::string name;
+    ///   std::string email;
+    ///   std::string phone;
+    /// };
+    /// struct Address
+    /// {
+    ///   int id;
+    ///   std::string address;
+    ///   std::string city;
+    ///   std::string country;
+    /// };
+    /// void example(DataMapper& dm)
+    /// {
+    ///   auto const sqlQueryString = R"(SELECT p.*, a.* FROM "Person" p INNER JOIN "Address" a ON p.id = a.id WHERE p.city =
+    ///   Berlin AND a.country = Germany)"; auto const records = dm.Query<Person, Address>(sqlQueryString); for (auto const&
+    ///   [person, address] : records)
+    ///   {
```

Comment:
```suggestion
    ///   auto const sqlQueryString = R"(SELECT p.*, a.* FROM "Person" p INNER JOIN "Address" a ON p.id = a.id WHERE p.city = Berlin AND a.country = Germany)";
    ///   auto const records = dm.Query<Person, Address>(sqlQueryString);
    ///   for (auto const& [person, address] : records)
    ///   {
```
Please proper statement composition (one per line)